plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestIcebergSnowflakeCatalogConnectorSmokeTest.java

```diff
@@ -684,6 +684,14 @@ public void testSnowflakeNativeTable()
                 .hasRootCauseMessage("SQL compilation error:\ninvalid parameter 'table ? is not a Snowflake iceberg table'");
     }
 
+    @Test
+    @Override
+    public void testIcebergTablesFunction()
+    {
+        assertThatThrownBy(super::testIcebergTablesFunction)
+                .hasMessageContaining("schemaPath is not supported for Iceberg snowflake catalog");
```

Comment:
Snowflake catalog supports `iceberg_tables` function, right? Skipping the entire test doesn't make sense to me. We could run this function without additional schema creation.

Reply:
@ebyhr Right, I did it quickly so the CI would be green + it follows a pattern in this test class.
I submitted a new [PR](https://github.com/trinodb/trino/pull/24593) that actually tests this for the snowflake catalog. PTAL